### PR TITLE
Fix outdated bundle vendor

### PR DESCRIPTION
**Pull request by quaff**
Jetty version will always be "9.4.z-SNAPSHOT" not exact "9.4.8.v20171121".

### CODE REVIEW

**Comment by janbartel** [comment]
@quaff  I don't understand the necessity for the changes here - looks like you've just changed the Implementation-Vendor entry in the manifest?  Can you explain what the problem is you want to fix?

I just ran jetty-9.4.12 release (from the distro) and I got the proper version as I expect:
>>2018-09-12 13:50:51.925:INFO:oejs.Server:main: jetty-9.4.12.v20180830; built: 2018-08-30T13:59:14.071Z; git: 27208684755d94a92186989f695db2d7b21ebc51; jvm 1.8.0_172-b11

**Comment by gregw** [comment]
@janbartel I'm guessing it has something to do with the bundle vendor in the jetty-all aggregate being "Eclipse Jetty Project" 
https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/aggregates/jetty-all/src/main/resources/META-INF/MANIFEST.MF#L4

**Comment by quaff** [comment]
Yes, I'm using jetty-all.

File: pom.xml [hunk 1]
```diff
@@ -307,7 +307,7 @@
                   <Bundle-ManifestVersion>2</Bundle-ManifestVersion>
                   <Bundle-Name>${project.name}</Bundle-Name>
                   <Bundle-SymbolicName>${bundle-symbolic-name}.source</Bundle-SymbolicName>
-                  <Bundle-Vendor>Eclipse.org - Jetty</Bundle-Vendor>
+                  <Bundle-Vendor>Eclipse Jetty Project</Bundle-Vendor>
```
Comment (joakime) [inline pom.xml]: Yeah, it probably should be called "Eclipse Jetty Project".
WDYT @jmcc0nn3ll  ?

Comment (gregw) [reply pom.xml]: That is what it is in most of our jars, so let's be consistent

**Pull request merged**